src/tools/dev/scripts/bv_support/bv_cgns.sh

```diff
@@ -88,128 +87,8 @@ function bv_cgns_dry_run
     fi
 }
 
-function apply_cgns_321_darwin_patch
-{
-    patch -p0 << \EOF
-diff -c cgnslib_3.2.1/src/configure.orig cgnslib_3.2.1/src/configure
-*** cgnslib_3.2.1/src/configure.orig    2015-04-27 15:11:36.000000000 -0700
---- cgnslib_3.2.1/src/configure 2015-04-27 14:24:48.000000000 -0700
-***************
-*** 2324,2333 ****
-  echo "$ac_t""$shared" 1>&6
-  
-  if test $shared = all; then
-!   exts="so sl a"
-    shared=yes
-  else
-!   exts="a so sl"
-  fi
-  if test $shared = yes; then
-    cgnsdir=`pwd`
---- 2324,2333 ----
-  echo "$ac_t""$shared" 1>&6
-  
-  if test $shared = all; then
-!   exts="dylib so sl a"
-    shared=yes
-  else
-!   exts="dylib a so sl"
-  fi
-  if test $shared = yes; then
-    cgnsdir=`pwd`
-***************
-*** 2352,2363 ****
-        shared=no
-      else
-        CFGFLAGS="-fPIC $CFGFLAGS"
-!       AR_LIB="\$(CC) -shared $SYSCFLAGS -Wl,-rpath,$LIBDIR:$cgnsdir/$BUILDDIR -o"
-!       EXT_LIB=so
-      fi
-    fi
-    if test $shared = yes; then
-!     RAN_LIB="\$(STRIP)"
-    fi
-  fi
-  
---- 2352,2363 ----
-        shared=no
-      else
-        CFGFLAGS="-fPIC $CFGFLAGS"
-!       AR_LIB="\$(CC) -shared $SYSCFLAGS -Wl,-L$with_hdf5/lib -Wl,-lhdf5 -o"
-!       EXT_LIB=dylib
-      fi
-    fi
-    if test $shared = yes; then
-!     RAN_LIB="\$(STRIP) -x"
-    fi
-  fi
-
-EOF
-    if [[ $? != 0 ]] ; then
-        return 1
-    fi
-
-    return 0
-}
-
-function apply_cgns_321_zlib_patch
-{
-    patch -p0 << \EOF
-diff -c cgnslib_3.2.1/src/configure.orig cgnslib_3.2.1/src/configure
-*** cgnslib_3.2.1/src/configure.orig    2013-06-19 21:04:00.000000000 -0700
---- cgnslib_3.2.1/src/configure 2015-04-27 15:03:16.000000000 -0700
-***************
-*** 2490,2495 ****
---- 2490,2496 ----
-  if test "${with_zlib+set}" = set; then
-    withval="$with_zlib"
-    withzlib=$withval
-+   ZLIBLIB=$withval
-  else
-    withzlib="no"
-  fi
-***************
-*** 2499,2504 ****
---- 2500,2506 ----
-    else
-      H5NEEDZLIB=1
-      if test -z "$withzlib" || test "$withzlib" = "yes"; then
-+       ZLIBLIB=""
-        zlibdir=""
-        echo "$ac_t""yes" 1>&6
-        ac_safe=`echo "zlib.h" | sed 'y%./+-%__p_%'`
-EOF
-    if [[ $? != 0 ]] ; then
-        return 1
-    fi
-
-    return 0
-}
-
-function apply_cgns_321_patch
-{
-
-    if [[ "$OPSYS" == "Darwin" ]] ; then
-        info "Applying OS X patch . . ."
```

Comment:
I take it these patches are no longer needed?

Reply:
I have no way to test without them, but the configure script has changed significantly in 6 years, so keeping these patches would probably introduce more issues than removing them.